chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -159,6 +159,7 @@
 if node[:platform_family] == "suse"
```

Comment:
@jgrassler this is the 2nd commit. The first commit is only useful for non-SUSE systems